### PR TITLE
[camera_web] Make plugin publishable for the first time.

### PR DESCRIPTION
**Pull request by ditman**
This PR gets the `camera_web` package ready to publish into pub.dev, as v0.2.0

(Partially addresses https://github.com/flutter/flutter/issues/45297)

* Clarify in the README that the plugin is not endorsed yet.
* Tweak the list of not-supported operations
* Call this v0.2.0

Thanks @bselwe!!

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Note that unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I [updated pubspec.yaml](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates) with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code

### CODE REVIEW

**Comment by ditman** [comment]
(Heads up @mvanbeusekom, @BeMacized; web support is finally coming!)

**Comment by ditman** [comment]
```
ERROR: * line 16, column 1 of lib/src/camera_web.dart: This package does not have stream_transform in the `dependencies` section of `pubspec.yaml`.
```

Nice check! I'll fix it right away! (I wish it was part of the analyzer checks, and not something on pre-publish)

File: packages/camera/camera_web/README.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ The web implementation of [`camera`][camera].
 
 ## Usage
 
-This package is [endorsed](https://flutter.dev/docs/development/packages-and-plugins/developing-packages#endorsed-federated-plugin), which means you can simply use `camera` normally. This package will be automatically included in your app when you do.
+### Depend on the package
+
+This package is not an endorsed implementation of the google_maps_flutter plugin yet,
```
Comment (stuartmorgan) [inline packages/camera/camera_web/README.md:11]: Optional nit: Maybe keep the link from the old version, but on "an endorsed implementation"?

**Review by stuartmorgan — APPROVED** [review]
LGTM! Please update the PR description to s/0.6.0/0.2.0/ before landing though.

File: packages/camera/camera_web/README.md (hunk 1 above)
Comment (ditman) [reply packages/camera/camera_web/README.md]: Done, I grabbed the text from another plugin that doesn't have the link (maps?), but it's a very good addition.

**Comment by ditman** [comment]
> Please update the PR description to s/0.6.0/0.2.0/ before landing though.

@stuartmorgan I think I scrubbed all the references to 0.6.0, thanks for the reminder!

**Pull request merged**